Neos.Flow/Classes/Cli/Response.php

```diff
@@ -118,10 +118,10 @@ public function getContent(): string
     /**
      * Sets color support / styled output to yes, no or auto detection
      *
-     * @param boolean $colorSupport true, false or NULL (= autodetection)
+     * @param boolean|null $colorSupport true, false or NULL (= autodetection)
      * @return void
      */
-    public function setColorSupport(bool $colorSupport): void
+    public function setColorSupport(bool|null $colorSupport): void
```

Comment:
Already has a type hint so I don't see how anyone would use it with NULL at this point and no one has missed it, I think it would be better to remove the reference to NULL from the doc block. We might consider deprecating this altogether and just auto detecting even.

Reply:
Yes one part of me agrees and i initially thought so too. But the whole class is build to allow `$colorSupport` to be null, and it is the default. This reset is unlikely to be used - especially as the `setColorSupport` is not even used currently - but most of our cli stuff should probably refactored and i just wanted to restore it to a consistent state.

Reply:
yeah fine by me.